app/ports/templates/ports/stats.html

```diff
@@ -182,7 +182,7 @@ <h6 class="text-center">Total Installations</h6>
                 </div>
                 <div class="col-lg-6 col-md-6 p-2">
                     <h6 class="text-center">Requested Installations</h6>
-                    <table class="table table-condensed border">
+                    <table class="table table-striped table-condensed border">
```

Comment:
What does this do?

Reply:
We received a feedback about using tables with rows having alternate colours. `table-striped` does that. But these tables were removed altogether, later.

Reply:
Oh, sorry, thanks. I was confusing striped vs. stripped.